<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Reinitialisation</class>
 <widget class="QDialog" name="Reinitialisation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Réinitialisation du mot de passe</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_question">
     <property name="text">
      <string>Question secrète affichée ici</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_reponse">
     <property name="placeholderText">
      <string>Votre réponse</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_nouveau_mdp">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Nouveau mot de passe</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_valider">
     <property name="text">
      <string>Valider</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
